src/containers/CreateAccount/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {FC, ReactNode, useState} from 'react';
+import axios from 'axios';
+
+import {AuthContainer} from 'components';
+import {Form, FormButton, FormInput} from 'components/FormComponents';
+import {formatAPIError} from 'utils/errors';
+import {standardHeaders} from 'utils/requests';
+import yup from 'utils/yup';
+
+const initialValues = {
+  confirmPassword: '',
+  email: '',
+  password: '',
+};
+
+export type FormValues = typeof initialValues;
+
+const CreateAccount: FC = () => {
+  const [creatingAccount, setCreatingAccount] = useState(true);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [submitting, setSubmitting] = useState<boolean>(false);
+
+  const handleSubmit = async ({email, password}: FormValues): Promise<void> => {
+    try {
+      setErrorMessage('');
+      setSubmitting(true);
+      await axios.post(`${process.env.REACT_APP_BACKEND_API}/users`, {email, password}, standardHeaders());
+      setCreatingAccount(false);
+    } catch (error) {
+      setErrorMessage(formatAPIError(error));
+    } finally {
+      setSubmitting(false);
+    }
+  };
+
+  const renderAuthContainerContent = (): ReactNode => {
+    return creatingAccount ? (
+      <Form initialValues={initialValues} onSubmit={handleSubmit} validationSchema={validationSchema}>
+        <FormInput label="Email" name="email" placeholder="" />
+        <FormInput label="Password" name="password" placeholder="" type="password" />
+        <FormInput label="Confirm Password" name="confirmPassword" placeholder="" type="password" />
+        <FormButton submitting={submitting} type="submit">
+          Create Account
+        </FormButton>
+      </Form>
+    ) : (
+      <p>
+        Our email verification system is still in development. Please contact Bucky on Slack to manually verify your
+        email.
+      </p>
+    );
+  };
+
+  const validationSchema = yup.object().shape({
+    confirmPassword: yup.string().oneOf([yup.ref('password'), ''], 'Passwords must match'),
+    email: yup.string().email().required('Email is required'),
+    password: yup.string().required('Password is required'),
```

Comment:
should we also include the password strength regex here so that the user knows the password strength criteria while typing in their password? currently, the criteria only shows after the user issues the post request:
![create](https://user-images.githubusercontent.com/32864116/103392594-a48feb80-4b59-11eb-9fd9-a281aa2d7c94.gif)

Reply:
There are some rules that are difficult to program into the frontend such as the `UserAttributeSimilarityValidator` and `CommonPasswordValidator` https://docs.djangoproject.com/en/3.1/topics/auth/passwords/ - I wasn't sure how to handle those and didn't want our FE and Be to get out of sync so I just let the BE throw back the errors.